src/main/java/core/basesyntax/Figure.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
-public abstract class Figure implements AreaCalculator, FigurePrinter {
-    protected String color;
+public abstract class Figure implements AreaCalculator, Printer {
+    private String color;
```

Comment:
Lets make it of type `Color` since we already have an enum for it, its is preferred to use it instead of String